src/css/support.js

```diff
@@ -76,6 +76,14 @@ define([
 				computeStyleTests();
 			}
 			return reliableMarginRightVal;
+		},
+
+		reliableMarginLeft: function() {
+			// Support: IE <=8 only, Android 4.0 - 4.3 only, Firefox <=3 - 37
```

Comment:
Yeah, it seems we didn't had consensus about this issue, it did come down to like/don't like, i'd say we should follow some standart here, if we so nit-picky :-).

http://practicaltypography.com/hyphens-and-dashes.html 
http://en.wikipedia.org/wiki/Dash#Ranges_of_values